### ISSUE TITLE
Request to add a double tap back function to exit the application

### ISSUE THREAD

**Issue by mak7im01** [zhanghai/MaterialFiles#1025]
Hello. Please add a double tap back function to exit the application. There are frequent closures without confirming the exit, which in turn is inconvenient.

**Here is an example of how I see this function:**
![Screenshot 03](https://github.com/zhanghai/MaterialFiles/assets/99025360/cb8f8054-f9ad-42b7-8f9c-d20bc18347d7)

**Comment by vishal2376** [comment]
I want to work on this issue. Can anyone assign it to me?

**Comment by zhanghai** [comment]
No, I'm not accepting PRs for now for reasons in the pull request template (despite that anyone can fork the project and maintain their fork). I can work on this in v1.6.3 though, since it's relatively simple and can be an opt-in setting.
